rskj-core/src/main/java/co/rsk/scoring/InetAddressUtils.java

```diff
@@ -40,11 +39,11 @@ public static boolean hasMask(String text) {
      *
      * @return  the text converted to an InetAddress
      */
-    public static InetAddress getAddressForBan(String name) throws InvalidInetAddressException {
-        if (name == null)
+    public static InetAddress getAddressForBan(String hostname) throws InvalidInetAddressException {

```

Comment:
Maybe we can annotate the `hostname` parameter with `@CheckForNull`, just for the sake of being future-proof :)